dependencies.yaml

```diff
@@ -144,6 +144,7 @@ dependencies:
         packages:
           - setuptools>=64.0.0
           - tomli
+          - scikit-build-core[pyproject]>=0.7.0
```

Comment:
I don't think that `distributed-ucxx` uses `scikit-build-core`. See the `pyproject.toml`:
```toml
[build-system]
build-backend = "setuptools.build_meta"
```

Even if it did, `package[extras]` entries can't be used in conda. See `build_python_ucxx` above, and you'll notice that only `output_types: [requirements, pyproject]` use the `package[extras]` form.

Reply:
This package probably needs to do the same as `dask_cudf`:

https://github.com/rapidsai/cudf/blob/241825a7b4db1713f44d8c298e08364b1eea9a32/python/dask_cudf/pyproject.toml#L60-L61

Reply:
You're quite right. I didn't realize `setuptools` had a utility to just read the version from a file.